modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/util/Utils.java

```diff
@@ -514,9 +520,59 @@ public static List<NameValuePair> getConsentRequiredClaimsFromResponse(HttpRespo
                 }
             }
         }
+
+        if (urlParameters.isEmpty()) {
+            urlParameters = extractConsentRequiredClaimsFromConsentPage(redirectUrl);
```

Comment:
Will this be true since even in the modified flow we will have the session data key as a url param?